client/src/main/java/com/paypal/selion/platform/mobile/android/UiSlider.java

```diff
@@ -0,0 +1,67 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2016 PayPal                                                                                          |
+|                                                                                                                     |
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.platform.mobile.android;
+
+import com.paypal.selion.logger.SeLionLogger;
+import com.paypal.selion.platform.grid.SeLionAppiumAndroidDriver;
+import com.paypal.selion.platform.mobile.elements.MobileSlider;
+import com.paypal.test.utilities.logging.SimpleLogger;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+
+/**
+ * <code>UiSlider</code> represents a Slider widget for Android UI automation framework.
+ */
+public class UiSlider extends UiObject implements MobileSlider {
+    private static final SimpleLogger logger = SeLionLogger.getLogger();
+    private static final int MIN_END_X = 1;
+    private static final int VALUE_UPPER_LIMIT = 1;
+
+    public UiSlider(String locator) {
+        super(locator);
+    }
+
+    /**
+     * it is not accurate and is best to used only for setting value to 0 or 1, otherwise the result is close to number
+     * @param value The desired decimal value from 0 to 1, inclusive. A 0 value represents far left and a value of 1
```

Comment:
@alizelzele this comment seems incomplete. Will you please update it.. A value of 1, far right?